save.go

```diff
@@ -92,11 +92,36 @@ func saveMain(_ *cobra.Command, args []string) error {
 	for _, lib := range libs {
 		libSaveDir := filepath.Join(savePath, unvendor(lib.Name()))
 		// Detect what type of license this library has and fulfill its requirements, e.g. copy license, copyright notice, source code, etc.
-		_, licenseType, err := classifier.Identify(lib.LicensePath)
+		licenseList, err := classifier.Identify(lib.LicensePath)
 		if err != nil {
 			return err
 		}
-		switch licenseType {
+
+		mostRestrictive := licenses.Unknown
```

Comment:
Should unknown be the most restrictive？ because it results in an error. No matter what you do, you cannot be sure you are conformant to the license.

Reply:
Also, the code here should better be extracted to the licenses folder.

Reply:
@Bobgy I fixed these issues in the latest commit.